File: 0136-single-number/0136-single-number.c
int singleNumber(int* a, int n) {
    int count = 0;
    for(int i=0;i<n;i++){
        count =0;
        for(int j=0;j<n;j++){
            if(a[i]==a[j]){
               count++;

            }
        }
            if(count ==1){
                return a[i];
            }
        
    }
    return 0;
}

File: 0136-single-number/0136-single-number.java
class Solution {
    public int singleNumber(int[] a) {
        /*int ans=0;
        for(int i=0;i<nums.length;i++){
            ans=ans^nums[i];
        }
        return ans;*/
        Arrays.sort(a);
        int n = a.length;
        for(int i=0;i<n-1;i=i+2){
            if(a[i]!=a[i+1])
                return a[i];
        }
        return a[n-1];

    }
}